src/ZendCodingStandard/ruleset.xml

```diff
@@ -201,232 +70,823 @@
+         operators. -->
+    <!-- checked by PSR12.Operators.OperatorSpacing -->
+
+    <!-- 7. Closures -->
+
+    <!-- Closures MUST be declared with a space after the function keyword, and
+         a space before and after the use keyword. -->
+    <!-- The opening brace MUST go on the same line, and the closing brace MUST
+         go on the next line following the body. -->
+    <!-- There MUST NOT be a space after the opening parenthesis of the argument
+         list or variable list, and there MUST NOT be a space before the closing
+         parenthesis of the argument list or variable list. -->
+    <!-- In the argument list and variable list, there MUST NOT be a space
+         before each comma, and there MUST be one space after each comma. -->
+    <!-- Closure arguments with default values MUST go at the end of the
+         argument list. -->
+    <!-- Argument lists and variable lists MAY be split across multiple lines,
+         where each subsequent line is indented once. When doing so, the first
+         item in the list MUST be on the next line, and there MUST be only one
+         argument or variable per line. -->
+    <!-- When the ending list (whether of arguments or variables) is split
+         across multiple lines, the closing parenthesis and opening brace MUST
+         be placed together on their own line with one space between them. -->
+    <!-- checked by Squiz.Functions.MultiLineFunctionDeclaration -->
+
+    <!-- If a return type is present, it MUST follow the same rules as with
+         normal functions and methods; if the use keyword is present, the colon
+         MUST follow the use list closing parentheses with no spaces between
+         the two characters. -->
+    <!-- checked by PSR12.Functions.ReturnTypeDeclaration -->
+
+    <!-- Additional Zend Framework rules -->
+
+    <!-- Inherited variables passed via `use` MUST be used in closures. -->
+    <rule ref="SlevomatCodingStandard.Functions.UnusedInheritedVariablePassedToClosure"/>
+
+    <!-- 8. Anonymous Classes -->
+
+    <!-- Anonymous Classes MUST follow the same guidelines and principles as
+         closures in the above section. -->
+    <!-- The opening brace MAY be on the same line as the class keyword so long
+         as the list of implements interfaces does not wrap. If the list of
+         interfaces wraps, the brace MUST be placed on the line immediately
+         following the last interface. -->
+    <!-- checked by PSR12.Classes.AnonClassDeclaration -->
+
+    <!-- 9. Commenting and DocBlocks -->
+
+    <!-- Additional Zend Framework rules -->
+
+    <!-- DocBlocks and comments SHOULD only be used if necessary. Code SHOULD
+         be written so it explains itself. They MAY NOT start with `#` and MAY
+         NOT be empty. They MAY be omitted and SHOULD NOT be used for already
+         typehinted arguments, except arrays.
+
+         PHPDoc tags `@param`, `@throws` and `@return` SHOULD not be aligned or
```

Comment:
This is one that I know quite a few have disagreed with.

I personally do not like to align, as it often leads to extraneous changes if a parameter is added with a longer name that previous values, or a parameter name is changed to one of a different length. Additionally, I find it can push any meaningful descriptions to the next line - and if line wrapping happens, the alignment is a moot point anyways if we have a rule indicating a single level of indentation for multi-line descriptions. (If we don't then the amount of indenting can make the gymnastics of wrapping lines really difficult.)

Proponents, however note that it becomes a more tabular format, which can be easier to read at the source code level.

@Ocramius — do you have opinions on this?

Reply:
I need to check this because I'm not sure if there is actually a sniff included to enforce/fix this. For a code review point of view I'd say don't align. For readability yes.

I've checked the doctrine code standard and I don't see anything mentioning this.